src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/deployment_map.py

```diff
@@ -83,10 +92,8 @@ def _get_all(self):
         )
         if not self.map_contents['pipelines']:
             LOGGER.error(
-                "No Deployment Map files found, create a deployment_map.yml "
-                "file in the root of the repository to create pipelines. "
-                "You can create additional deployment maps if required in a "
-                "folder named deployment_maps with any name (ending in .yml)"
+                "No Deployment Map files found, create a deployment_map.yml file in the root of the repository to create pipelines. "
+                "You can create additional deployment maps if required in a folder named deployment_maps with any name (ending in .yml)"
```

Comment:
Can you revert this change, so the line length matches what we defined in the .editorconfig file?